### PR TITLE
feat: `create-onchain`

### PR DESCRIPTION
**Pull request by 0xAlec**
**What changed? Why?**
- add `create-onchain` CLI package

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: create-onchain/templates/next/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-ock`]().
```
Comment (fakepixels) [inline create-onchain/templates/next/README.md:1]: @0xAlec should this be "create onchain"?

File: create-onchain/src/cli.ts [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+            )} (optional)`
+          ),
+        },
+        {
+          type: 'toggle',
+          name: 'smartWallet',
+          message: pc.reset('Use Coinbase Smart Wallet? (recommended)'),
+          initial: true,
+          active: 'yes',
+          inactive: 'no',
```
Comment (fakepixels) [inline create-onchain/src/cli.ts:118]: Love this!

File: create-onchain/templates/next/app/page.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  {
+    name: 'Transaction',
+    url: 'https://onchainkit.xyz/transaction/transaction',
+  },
+  { name: 'Swap', url: 'https://onchainkit.xyz/swap/swap' },
+  { name: 'Pay', url: 'https://onchainkit.xyz/pay/pay' },
+  { name: 'Wallet', url: 'https://onchainkit.xyz/wallet/wallet' },
+];
+
+const templates = [
```
Comment (fakepixels) [inline create-onchain/templates/next/app/page.tsx:32]: Love this!!

File: create-onchain/src/cli.ts (hunk 2 above)
Comment (fakepixels) [reply create-onchain/src/cli.ts:118]: Think we can add theming too once that work is done.

**Review by fakepixels — APPROVED** [review]

**Review by fakepixels — APPROVED** [review]

**Comment by cpcramer** [comment]
Could you add a PR description and possibly include images or videos to help with reviewing this change? 🙏

**Pull request merged**